vmdb/db/fixtures/ae_datastore/ManageIQ/Service/Provisioning/StateMachines/Methods.class/__methods__/catalogbundleinitialization.rb

```diff
@@ -1,131 +1,120 @@
+# CatalogBundleInitialization.rb
 #
+# Author: Kevin Morey <kmorey@redhat.com>
+# License: GPL v3
 #
-# Description: This method sets the dialog optioms in the destination service and service catalog items.
-# 1. Look for all Service Dialog Options in the service_template_provision_task.dialog_options
-#    (i.e. Options that come from a Service Dialog)
-# 2. Any Service dialog option keys that match the regular expression /^tag_0_(.*)/i will be used to
-#    tag the destination Service
-# 3. Service dialog option keys that match the regular expression /^(option|tag)_(\d*)_(.*)/
-#    (I.e. <option_type>_<sequence_id>_variable,  option_0_vm_memory, tag_1_environment) are sorted
-#    by sequence_id into an options_hash[sequence_id] and then distributed to the appropriate
-#    child catalog items. For example, a sequence_id of 0 means that all catalog items will inhereit
-#    these variables. A sequence_id of 1 means that the variable is only intended for a catalog item
-#    with a group index of 1.
-# 4. Service dialog option keys that do NOT match the regular expression are then inserted into the
-#    options_hash[0] for all catalog items.
-#
-# Inputs: $evm.root['service_template_provision_task'].dialog_options
+# Notes: This method Performs the following functions:
+# 1. YAML load the Service Dialog Options from @task.get_option(:parsed_dialog_options))
+# 2. Set the name of the service
+# 3. Set tags on the service
+# 5. Pass down any dialog options and tags to child catalog items pointing to CatalogItemInitialization
 #
+begin
+  def log(level, msg, update_message = false)
```

Comment:
@tinaafitz 
This function name could be log_and_update_message instead of log since it is doing 2 things.